cli/src/main.rs

```diff
@@ -264,9 +266,8 @@ fn run() -> Result<(), CliError> {
                     .required_unless("node_file")
                     .multiple(true)
                     .long_help(
-                        "Node that is part of the circuit. \
-                         Format: <node_id>::<endpoint>. \
-                         Endpoint is optional if node alias has been set.",
+                        "Node that is part of the circuit \
+                         Format: <node_id>::<endpoint>",
```

Comment:
These `Format` and `Default value` help strings are not printed with new lines. This one shows up as: `Node that is part of the circuit Format: <node_id>::<endpoint>`. Maybe a better format is `Description of argument (format: xxx::yyy)`.

Reply:
Reformatting to @vaporos's suggestion `(xxx::yyy)`